src/intro.md

```diff
@@ -237,6 +238,7 @@ Keep lines sorted.
 [ex-std-read-lines]: basics.html#ex-std-read-lines
 [ex-tar-compress]: app.html#ex-tar-compress
 [ex-tar-decompress]: app.html#ex-tar-decompress
+[ex-tar-decompress-remove-path-prefix]: app.html#ex-tar-decompress-remove-path-prefix
```

Comment:
please use shorter example tag something along the lines of `ex-tar-strip-prefix`